controller/rancher/rancher.go

```diff
@@ -173,6 +200,28 @@ func (lbc *LoadBalancerController) ScheduleApplyConfig(string) {
 	lbc.syncQueue.Enqueue(lbc.GetName())
 }
 
+func (lbc *LoadBalancerController) IsEndpointUpForDrain(ep *config.Endpoint) bool {
+	if lbc.LBProvider.IsEndpointUpForDrain(ep) {
+		logrus.Info("DrainEndpoint: The endpoint is already in drainlist")
```

Comment:
Seems excessive to log this at Info, doesn't it?
Also, seems like we should log soemthing to know what endpoint we're talking about, like the ip or whatever useful info we have in ep

Reply:
Same applies to below log statements too, but if you think some/all of those should still be info, i'll buy that.

Reply:
yes, removed these. had them in for tests